.github/workflows/e2e-tests.yml

```diff
@@ -0,0 +1,125 @@
+name: E2E Tests for PR
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    paths-ignore:
+      - "docs/**"
+      - "**.md"
+      - ".circleci/**"
+
+jobs:
+
+  build:
+    if: github.event.pull_request.draft == false
+    runs-on: buildjet-2vcpu-ubuntu-2004
+    timeout-minutes: 25
+    strategy:
+      matrix:
+        edition: [oss, ee]
+    env:
+      MB_EDITION: ${{ matrix.edition }}
+      INTERACTIVE: false
+    steps:
+    - uses: actions/checkout@v3
+    - name: Prepare front-end environment
+      uses: ./.github/actions/prepare-frontend
+    - name: Prepare back-end environment
+      uses: ./.github/actions/prepare-backend
+
+    - run: ./bin/build version
+    - run: ./bin/build translations
+    - run: ./bin/build frontend
+    - run: ./bin/build licenses
+    - run: ./bin/build drivers
+    - run: ./bin/build uberjar
+
+    - name: Prepare uberjar artifact
+      uses: ./.github/actions/prepare-uberjar-artifact
+
+  e2e-tests:
+    runs-on: buildjet-4vcpu-ubuntu-2004
```

Comment:
The` if: github.event.pull_request.draft == false` is only for the build Job, shouldn't we have it here as well ?

Reply:
No need, because of the `needs:` key.